src/server.js

```diff
@@ -44,6 +45,8 @@ const server = http.createServer(app);
 // serve the static HTML
 app.use(express.static(STATIC_DIR));
```

Comment:
this still serves `/wallet`. please make that redirect to the terminal slash so it's consistent.

Reply:
<img width="739" alt="Screen Shot 2022-10-26 at 8 37 00 AM" src="https://user-images.githubusercontent.com/8848650/198071148-c692e7c3-3c85-4af4-90dd-82ec069ad3a5.png">

Reply:
> if I visit http://localhost:3000/wallet it becomes http://localhost:3000/wallet/ 
ah, I didn't realize that. yeah that's fine. What's doing that?

Reply:
I think that's just how `express.static` behaves if you access a directory path instead of a file path. It adds a trailing slash and looks for an `index.html` to serve